lib/raygun.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
+
+const env = require('env-var');
+const raygun = require('raygun');
+
+module.exports = new raygun.Client().init({
```

Comment:
I believe you should actually be using https://github.com/TkeITMobility/rhmap-raygun-nodejs

It also assumes the env var RAYGUN_API_KEY if you don't provide one, so you can just provide an empty object `{}` to the config.